lib/util/print.js

```diff
@@ -91,4 +96,25 @@ function printTreeTraversal(node) {
   }
 }
 
+// Need to add parens around any [+, -] operation
+// next to [/, *, ^] operation or parenthesis node
+// Examples:
+// { "/" [{ "+" ["x", "2"] }, "2"] } -> (x + 2) / 2.
+// { "+" [{ "+" ["x", "x"] }, { "+" ["2", "2"] }] } -> (x + x) + (2 + 2)
+// Check #120, #126 issues for more details.
+function needToAddParens(node, parentNode) {
+  if (!parentNode ||
+      !Node.Type.isOperator(parentNode) ||
+      '*/^'.indexOf(node.op) !== -1) {
+    return false;
+  }
+
+  if ('+-'.indexOf(parentNode.op) === -1 ||
```

Comment:
same here, `'*/^'.indexOf(parentNode.op) >= 0` shows what we're looking for instead of what we're not looking for, which I find easier to follow